docs/source/architecture.rst

```diff
@@ -0,0 +1,143 @@
+==================================
+
+The Drivers Atlas Testing project validates the behavior of drivers against
+Atlas clusters. For example, a test could be to perform queries or writes
+in a loop while Atlas performs a maintenance operation, and assert that all
+queries and writes complete successfully.
+
+The project consists of the following principal components:
+
+- Astrolabe - the orchestration utility.
+- :doc:`Workload executors <spec-workload-executor>` - wrappers providing
+  a common interface to each driver's unified test runner.
+- Each driver's unified test runner.
+- :doc:`The test scenario specification <spec-test-format>`.
+
+Astrolabe
+---------
+
+Users invoke ``astrolabe`` to perform a test. Astrolabe interacts with Atlas
+and it performs several common tasks to keep the workload executors as small
+and simple as possible. Specifically, Astrolabe:
+
+- Creates an Atlas cluster for testing.
+- Launches a workflow executor pointed at the cluster.
+- Initiates Atlas maintenance operations in accordance with the test scenario.
+- Waits for all operations to complete.
+- Requests termination of the workflow executor.
+- Verifies the successful termination of the workflow executor
+- Verifies that the workflow executor executed the workload.
+- Retrieves Atlas logs and FTDC data for problem diagnosis.
+- Calculates statistics and aggregates information obtained during the
+  test run, for example it calculates the peak number of connections per
+  server and average operation latency.
+- Tears down the Atlas cluster.
+- Uploads Atlas logs, event data produced by the workflow executor and
+  aggregated statistics as Evergreen build artifacts.
+
+Astrolabe is:
+
+- Written in Python.
+- Runs on Linux and Windows.
+- Provides a CLI interface.
+- Is the only component of Drivers Atlas Testing that communicates with the
+  Atlas API.
+
+
+Workload Executors
+------------------
+
+Each driver that is tested by Drivers Atlas Testing must implement a
+:doc:`workload executor <spec-workload-executor>`,
+which is a wrapper script that essentially provides a common
+interface to the driver's unified test runner.
+
+The workload executor is invoked by Astrolabe once a test Atlas cluster is
+up and running. The workload executor:
+
+- Instantiates the driver's unified test runner.
+- Receives the workload specification, which is a unified test format test,
+  from Astrolabe, and passes it to the unified test runner.
+- Sets up a termination signal handler.
+- Invokes the unified test runner to execute the workload, which ordinarily
+  contains a ``loop`` operation.
+- When the termination signal is received from Astrolabe, instructs the
+  unified test runner to stop looping.
+- Collects the number of iterations and successful operations performed by
+  the test runner, the number of errorsr and failures, the events produced
+  by the driver while executing the workload, and writes all of this
+  to files.
+
+
+Unified Test Runners
```

Comment:
I think with the changes I have suggested above, this section can be safely removed. IMHO we are better off without it since this (the unified format) exists independently from D-A-T (even though features have been added to it for D-A-T purposes).